main/menu.ts

```diff
@@ -20,14 +20,28 @@ export default function setMenu(win: Electron.BrowserWindow) {
                     label: 'Undo',
                     accelerator: 'CmdOrCtrl+Z',
                     click: () => {
-                        win.webContents.send('nyaovim:exec-commands', ['undo']);
+                        const focusedWebContents = webContents.getFocusedWebContents();
+                        if (win.webContents === focusedWebContents) {
```

Comment:
We can use `isDevToolsFocused` for this purpose.

```typescript
if (win.webContents.isDevToolsFocused()) {
  // send to nvim
} else {
  // default behavior
}
```

https://github.com/electron/electron/blob/master/docs/api/web-contents.md#contentsisdevtoolsfocused